Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+
+# Install dependencies.
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     python python3-pip python3-tk texlive-latex-base latexmk git vim locales
```

Comment:
Can get rid of `python` if using `FROM python:3`. Where do we use `python3-tk`?  Forgot `emacs` :)

Reply:
python3-tk was listed as a prerequisite.